test/testLiveAsTemplate.js

```diff
@@ -124,6 +124,10 @@ describe('CLI interface, oghliner as a template', function() {
     .then(function() {
       assert.doesNotThrow(fse.statSync.bind(fse, 'index.html'));
       assert.doesNotThrow(fse.statSync.bind(fse, 'offline-worker.js'));
+    })
+    .catch(function(err) {
+      console.log('Error: ' + err);
+      throw err;
```

Comment:
I did a quick test, throwing an error in the previous link of the chain, and it didn't display the message twice, although it did display the word "Error" twice, i.e. `Error: Error: text of error message`. Presumably this is because `new Error('text of error message').toString()` produces `Error: text of error message`, so `console.log('Error: ' + err)` duplicates the `Error:` prefix.